app/actions/commute.js

```diff
@@ -0,0 +1,55 @@
+import _ from 'lodash';
+import directions from '../services/directions';
+
+export const updateCommutes = (dispatch, getState) => {
+  const state = getState();
+
+  const houseIds = Array.from(state.houses.keys());
+  const destinationIds = Array.from(state.destinations.keys());
+  const knownCommutes = Array.from(state.commutes.keys());
+
+  const desiredCommutes = destinationIds
+    .map((destinationId) => houseIds.map((houseId) => `${houseId}.${destinationId}`))
```

Comment:
I'm making a string here so that `_.difference` does the sensible thing by default, which is a bit sad but seems like the least sad way to do it